processors/md_processor.py

```diff
@@ -0,0 +1,68 @@
+import re
+from processors import CellProcessor
+
+
+class MdProcessor(CellProcessor):
+
+    def __init__(self, cell_data, nlp=None):
+        super().__init__(cell_data)
+
+        self.task_mapping = {
+            'cell_language': self.get_cell_language,
+            'sentences_count': self.get_sentences_count,
+            'unique_words': self.get_unique_words,
+            'content': self.get_md_content,
+        }
+
+        self.nlp = nlp
+
+    def process_cell(self, tasks) -> dict:
+        for function in {
+            k: v
+            for k, v in tasks.items()
+            if (v and k in self.task_mapping.keys())
+        }:
+            self.cell[function] = self.task_mapping[function](self.cell)
+        return self.cell
+
+    def get_cell_language(self, cell):
+        doc = self.nlp(cell['source'])
```

Comment:
This is very expesive operation. Can we cache the result of nlp (same could be done with ast in previous processor)